mlos_bench/mlos_bench/storage/sql/experiment.py

```diff
@@ -281,6 +279,13 @@ def pending_trials(self, timestamp: datetime, *, running: bool) -> Iterator[Stor
                     config=config,
                 )
 
+    def pending_trials(self, timestamp: datetime, *, running: bool) -> Iterator[Storage.Trial]:
```

Comment:
```suggestion
    def pending_trials(self, timestamp: datetime, *, running: bool) -> Iterator[Storage.Trial]:
```

I'm OK to get rid of this one and replace the others with `filter_trials_by_status` and feed it `Status.RUNNABLE_STATUSES` which returns that list as a literal or something.

Maybe separate that out to a different PR for cleanliness.